cookbook/profiler/data_collector.rst

```diff
@@ -203,7 +203,7 @@ must also define additional blocks:
             </div>
         {% endset %}
 
-        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig') }}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', {'link': true}) }}
```

Comment:
It should be `true` because this example requires access to profile panel. in the example before this is not required.